orange/src/main/java/com/hartwig/hmftools/orange/OrangeApplication.java

```diff
@@ -25,18 +26,21 @@ public class OrangeApplication {
     public static void main(String[] args) throws IOException {
         LOGGER.info("Running {} v{}", APPLICATION, VERSION);
 
-        Options options = OrangeConfig.createOptions();
+        ConfigBuilder configBuilder = new ConfigBuilder();
+        OrangeConfig.registerConfig(configBuilder);
+
+        if(!configBuilder.parseCommandLine(args))
+        {
+            configBuilder.logInvalidDetails();
+            System.exit(1);
+        }
 
-        OrangeConfig config = null;
         try {
-            config = OrangeConfig.createConfig(new DefaultParser().parse(options, args));
+            OrangeConfig config = OrangeConfig.createConfig(configBuilder);
+            new OrangeApplication(config).run();
         } catch (ParseException exception) {
-            LOGGER.warn(exception);
-            new HelpFormatter().printHelp(APPLICATION, options);
```

Comment:
Does the help text still get printed somewhere?

Reply:
Yes it does by the ConfigBuilder class